pkg/controllers/sgx/controller.go

```diff
@@ -110,7 +110,7 @@ func setInitContainer(spec *v1.PodSpec, imageName string) {
 			},
 			VolumeMounts: []v1.VolumeMount{
 				{
-					MountPath: "/etc/kubernetes/node-feature-discovery/source.d/",
+					MountPath: "/etc/kubernetes/node-feature-discovery/source.d/:z",
```

Comment:
https://github.com/intel/intel-device-plugins-for-kubernetes/issues/762#issuecomment-1014903292 suggests `:z` should not be used.

Reply:
ahh i see, it worked but its not supposed to and may not work in the future. thanks for pointing out. looks like we need to go back to using spc_t.

Reply:
See also https://github.com/intel/intel-device-plugins-for-kubernetes/issues/762#issuecomment-1024259524

Reply:
removed and updated with label for init containers from container-selinux project.